### DISCUSSION TITLE
Lutrisview: a simple bigpicture like mode for lutris

### DISCUSSION THREAD

**Discussion by redmie** [lutris/lutris#5608] [Show and tell]
Dear all,

I didn't manage to use lutris without keyboard/mouse input e.g. with only a controller.
Thus, I created [lutrisview](https://github.com/redmie/lutrisview) to browse my lutris library with a controller and get closer to a console like experience (screenshot underneath). Please check it out !

![lutrisview-by-last-played](https://github.com/user-attachments/assets/afadd9dd-9f0b-4d81-801c-55faece7815d)

This is a simple python script leveraging pygame installed as a linux desktop application. Internally it leverages lutris ability to output json on its command line interface and to launch game by ID.  It also leverages lutris cache folder to get game coverarts (the hackiest part ^^). This simplicity should make lutrisview quite robust against API changes.

Many thanks to all Lutris contributors!

Best regards

**Comment by strycore** [comment]
Very cool!

Please note that we have plans for a big picture UI with the use of OpenGamepadUI.  https://github.com/ShadowBlip/OpenGamepadUI/

While OpenGamepadUI already supports Lutris, we'd like to have a better integration with it and ship as a component of Lutris itself.

**Reply by monospacedmagic** [reply]
Since this is one of the top search results for "lutris big picture": Are there any updates on this?

**Reply by strycore** [reply]
There is a Lutris plugin for OGUI which is basic but functional.

**Comment by GloriousEggroll** [comment]
i acitually like this idea without opengamepadui. i gave opengamepadui a test drive and it seems it wants to be its own steam drop in replacement rather than just a game manager. right now both lutris and heroic have a hit or miss at best interface for big picture mode. it might be cool to have this as a lutris launch option, at least until a more permanent and/or planned out solution is in place
